api/src/main/resources/org/openmrs/module/ugandaemrreports/reports/METRICS.json

```diff
@@ -0,0 +1,17 @@
+{
+  "facilityid": "#S.dhis_uuid#",
```

Comment:
@dbaluku Add date_metrics_computed

Reply:
@ssmusoke the date the metrics were computed is auto-filled in the database on the server side